website/docs/r/cosmosdb_mongo_collection.html.markdown

```diff
@@ -53,7 +54,8 @@ The following arguments are supported:
 * `resource_group_name` - (Required) The name of the resource group in which the Cosmos DB Mongo Collection is created. Changing this forces a new resource to be created.
 * `database_name` - (Required) The name of the Cosmos DB Mongo Database in which the Cosmos DB Mongo Collection is created. Changing this forces a new resource to be created.
 * `default_ttl_seconds` - (Required) The default Time To Live in seconds. If the value is `-1` items are not automatically expired.
-* `shard_key` - (Required) The name of the key to partition on for sharding. There must not be any other unique index keys. 
+* `shard_key` - (Required) The name of the key to partition on for sharding. There must not be any other unique index keys.
+* `throughput` - (Optional) The throughput of MongoDB collection (RU/s). Must be set in increments of 100. Default value is 400.
```

Comment:
```suggestion
* `throughput` - (Optional) The throughput of the MongoDB collection (RU/s). Must be set in increments of 100. The default value is 400.
```

Reply:
If 400 is the minimum we should state that here to

Reply:
done